README.md

```diff
@@ -96,7 +97,22 @@ mvn orphedomos:multiplatform-build-deploy -Dorphedomos.build.architecture=linux/
   <targetArchitectures>linux/arm64</targetArchitectures>
   <targetArchitectures>linux/amd64</targetArchitectures>
 </targetArchitectures>
+
+
+```
+
+#### Push Docker Image
+
+This command leverages the Docker daemon to push the generated Docker image to a repository. In order to use this command. The generated Docker image must already be loaded into Docker. The command cannot be used directly on a Docker image tarball.
```

Comment:
S (Substantive): this is a little hard to follow.  Also I think you want a comma after command?

Reply:
Changed to have a simpler description.